libs/designsystem/action-group/src/action-group.component.ts

```diff
@@ -66,15 +67,30 @@ export class ActionGroupComponent implements AfterContentInit {
     return 'align-' + this.align;
   }
 
+  private collapseThreshold = 2;
+
   constructor(
     private renderer: Renderer2,
     @Optional() @Inject(ACTIONGROUP_CONFIG) private config: ActionGroupConfig
   ) {}
 
   ngAfterContentInit(): void {
+    // Ensure we collapse according to visibleActions if lower than our default threshold (2).
+    // I.e. if there are 2 buttons and visibleActions = 1 we'll collapse the 2nd button into the menu:
+    if (this.visibleActions < this.collapseThreshold) {
+      this.collapseThreshold = this.visibleActions;
+    }
     if (this.config) {
```

Comment:
This structure has quite some nested `if`s, could we, could this be inverted to an early return instead?